src/test/suite/fixtures/recorded/languages/clojure/changeCallee4.yml

```diff
@@ -0,0 +1,23 @@
+languageId: clojure
```

Comment:
huh this one's a bit of a corner case.  I'd almost expect to get `f` here, but I'm not sure; what you have may be more consistent.  Maybe let's get a couple second opinions cc/ @somanythings @wenkokke